### DISCUSSION TITLE
malloy binaries when python package is installed

### DISCUSSION THREAD

**Discussion by bernardro** [malloydata/malloy#2006] [Q&A]
when one does
```python
pip install malloy
```

three binaries are also installed under `malloy/service/`
> malloy-service-macos-x64
malloy-service-linux-x64
malloy-service-linux-arm64

where do these come from and how can I inspect them

context is that I am trying to get malloy to play nice with dagster and I have ported malloy-py code into my application so that I can debug it - then I came across those three

**Comment by gmaden** [comment]
The repo for those binaries is located at https://github.com/malloydata/malloy-service
